doc/sphinx/addendum/type-classes.rst

```diff
@@ -366,10 +366,10 @@ Summary of the commands
 
    .. deprecated:: 8.14
 
-      The default value for instance locality will change in a future
-      release. Instances added outside of sections without an explicit
-      locality are now deprecated. We recommend using :attr:`export`
-      where possible.
+      The default value for instance locality will change in a future release.
```

Comment:
Is it still a future change?  If not, drop the sentence.

Reply:
Yes, we only make the deprecation more discouraging in this PR. The default value will change next iteration.